pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1801,7 +1801,7 @@
       "BaselineVersion": "4.0.0",
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.0.0",
-        "4.0.1.0": "4.3.0"
+        "4.1.0.0": "4.3.0"
```

Comment:
I'm not sure what we're doing for package versions at this point.  Based on our current branch schedule should this be 4.3 or 4.4?